tests/e2e/testcases/subnet/local/suite.go

```diff
@@ -164,3 +167,81 @@ var _ = ginkgo.Describe("[Local Subnet]", func() {
 		commands.DeleteSubnetConfig(secondSubnetName)
 	})
 })
+
+var _ = ginkgo.Describe("[Subnet Compatibility]", func() {
+	ginkgo.AfterEach(func() {
+		commands.CleanNetwork()
+		err := utils.DeleteConfigs(subnetName)
+		if err != nil {
+			fmt.Println("Clean network error:", err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+		err = utils.DeleteConfigs(secondSubnetName)
+		if err != nil {
+			fmt.Println("Delete config error:", err)
+		}
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	ginkgo.It("can deploy a subnet-evm with old version", func() {
+		subnetEVMVersion := "v0.4.2"
```

Comment:
Couldn't these be quickly get outdated if they are hardcoded?

Reply:
for this particular test, that's actually the point